tests/test_kernelshap.py

```diff
@@ -16,7 +15,7 @@ def test_shap_segment_image(self):
         n_segments = 50
         compactness = 10.0
         sigma = 0
-        image_segments = explainer._segment_image(  # pylint: disable=protected-access
+        image_segments = explainer._segment_image(
```

Comment:
Here another very deserved complaint by pylint :) It's bad form to access "protected" functions. Sure, it is a test, in this case, but does it really make sense to have `_segment_image` protected? Usually, protected (or private with double underscore) functions are protected/private because they modify internal class/object properties that you don't want users to mess with. In this case that is totally not true: the function is static (so it changes nothing in objects) and in fact only passes along parameters to another, external function. There's no reason to keep it protected.

Note: I'm not trying to review the contents here, I'm just giving examples of pylint's justified complaining :D